WebRoot/js/ajax/dwt/widgets/DwtLabel.js

```diff
@@ -284,7 +284,10 @@ DwtLabel.prototype.setText = function(text) {
         this._textEl.innerHTML = text;
     }
 
-	this.setAriaLabel(text);
+	// Do not set combobox selected value as aria-label
+	if (!(this instanceof DwtSelect)) {
```

Comment:
instead of this check, can we override setAriaLabel method in DwtSelect and ignore the set call?

Reply:
There are few DwtSelect components where we need to set aria-label and We are setting aria-label for DwtSelect by calling `setAriaLabel` method. See this https://github.com/Zimbra/zm-web-client/pull/741/files#diff-4ed61d64a7de753d4d962054ced443cb4404e55131d1e45a515acc08f97972a0R588

So here we are preventing to set aria-label on calling setText method.